.github/workflows/trivy-scan.yml

```diff
@@ -106,6 +111,10 @@ jobs:
           # kapp-controller docker image - output in sarif and json
           trivy image --ignore-unfixed --format sarif --output trivy-results-image.sarif "docker.io/carvel/kapp-controller:${{ github.sha }}"
           trivy image --ignore-unfixed --format json --output trivy-results-image.json "docker.io/carvel/kapp-controller:${{ github.sha }}"
+
+          # kctrl binary - output in sarif and json
+          trivy rootfs --ignore-unfixed --format sarif --output trivy-results-kctrl.sarif "kctrl"
```

Comment:
My main concern is that the trivy scans will be mangled with kapp-controller, which can cause some issues, but to be fair I am not 100% sure if we have another solution. GitHub UI is not very good in this aspect

Reply:
100%, it almost feels like we need some other way to report these CVEs cause the github UI is just too hard to disentangle stuff.

Reply:
Yeah, I was trying to see if the vulnerabilities can somehow be grouped/labeled, but couldn't find anything 😞

Reply:
@neil-hickey Let me know if this is good for now or we should find an alternate solution.

P.S I added another commit to fix the recent failures happening for Trivy scan.

Reply:
This seems fine to me, my understanding is that we will shortly be moving off of trivy anyways. @joaopapereira feel free to correct me on that one. So for now, I think mixing it in is fine. LGTM!